backend/src/main/scala/sbt/internal/inc/bloop/internal/BloopHighLevelCompiler.scala

```diff
@@ -132,6 +147,13 @@ final class BloopHighLevelCompiler(
               config.progress.toOptional,
               logger
             )
+
+            withTee { tee =>
+              val result = baos.toString()
+              if (result.nonEmpty)
+                logger.info(baos.toString)
+              tee.removeListener(baos)
+            }
```

Comment:
Does it mean we only get the println at the end of the compilation? So what if the compiler crashes? Should we add a `try catch` around it?

Reply:
We should, otherwise might hard debugging compiler crashes

Reply:
Moved it to a finally block.